theme/styles/snippets/markdown.scss

```diff
@@ -0,0 +1,114 @@
+//Change hover link styling when designs are added.
+
+.Markdown {
+  padding-bottom: 3.125rem;
+    
+  @include media ('lg-up') {
+    padding-bottom: 12.5rem;
+  }
+
+  h1, h2, h3, p, ul, ol {
+    font-family: $itc-garamond;
+  }
+
+  h2 {
+    padding-bottom: 3.125rem;
+    text-transform: uppercase;
+    @extend %serif-xl;
+  }
+
+  ul, ol, p {
+    @extend %serif-sm;
+    padding-bottom: 2.875rem;
+
+    @include media('lg-up') {
+      font-size: 2.5rem;
+      line-height: 2.625rem;
+    }
+
+    strong {
+      font-weight: 700;
+    }
+
+    a {
+      @extend %transition-shorter;
+      &:hover {
+        color: lighten(color('black'), 25%);
+      }
+    }
+  }
+
+  ul,
+  ol {
+    list-style-position: inside;
+  }
+
+  ul {
+    list-style-type: none;
+
+    
+    li {
+      &::before {
+        content: "-";
+        margin-right: 0.625rem;
+      }
+    }
+  }
+
+  table { 
+    border-collapse: collapse; 
```

Comment:
@sepowitz I know it's not ideal to have the table be fully markdown since that could be prone to user error, but in sprint Hugh decided on markdown for the mvp so we can move faster! 😬